backend/src/main/java/ca/bc/gov/app/converters/SubmissionMatchDetailEntityBeforeConvert.java

```diff
@@ -34,13 +47,27 @@ public Publisher<SubmissionMatchDetailEntity> onBeforeConvert(
     return Mono.justOrEmpty(entity.withMatchingField(convertTo(entity)));
   }
 
+  /**
+   * This method is called after the entity is retrieved from the database.
+   * It converts the matchingField field of the entity from a JSON string to a Map.
+   *
+   * @param entity The entity that was retrieved from the database.
+   * @param table The table where the entity was retrieved from.
+   * @return The entity with the converted matchingField field.
```

Comment:
Same question, should we start @<!-- -->return with "A Publisher that emits ..."?

Reply:
That makes complete sense. Updated the docs